apps/src/music/MusicView.jsx

```diff
@@ -326,6 +346,27 @@ class UnconnectedMusicView extends React.Component {
     // This is a way to tell React to re-render the scene, notably
     // the timeline.
     this.setState({updateNumber: this.state.updateNumber + 1});
+
+    this.saveCode();
+  };
+
+  saveCode = () => {
+    const code = Blockly.blockly_.serialization.workspaces.save(this.workspace);
+    const codeJson = JSON.stringify(code);
+    localStorage.setItem('musicLabSavedCode', codeJson);
```

Comment:
Discussed this briefly over Slack, but how do you feel about putting this behind some generic saving/loading interface, that either talks to local storage or the projects system (or whatever backend we want)? Thinking about how we might make this more generic with lab lab (cc: @molly-moen @madelynkasula for thoughts as well). I'm okay to ship with this implementation, but just curious about the possibilities

Reply:
I'm in favor of separating concerns as much as possible to improve readability and decrease the work to refactor this into lab lab. But if we are still considering this a prototype without a goal of full integration into the system it's ok to leave it as-is.

Reply:
All good points.  I think it will be useful to make this improvement soon.  It seems like our interface should support asynchronous operations, and simulate network delays for saving and loading, and the app should throttle or debounce the saves, and handle the delayed response for a load.